### PR TITLE
rc5: unlock parameter size, add u128 and u8 word size support.

### PR DESCRIPTION
**Pull request by NamedNeon**
Solves #381.

- [x] Unlock RC5 object sizes (key size, word size, etc.).
- [x] Add compatibility for more word sizes.
- [x] Increase test and benchmark coverage for more common sizes.

### CODE REVIEW

**Comment by NamedNeon** [comment]
This is ready for review now. 

I unintentionally set it as ready last night and didn't realize until this morning, so apologies for that. I don't make pull requests very often.

File: rc5/CHANGELOG.md [hunk 1]
```diff
@@ -7,3 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.0.1 (2023-02-10)
 - Initial release
+
+## 0.1.0 (2023-08-30)
+- Added u8 and u128 word size support.
+- Enabled custom word size, key size, and round count values.
+- Deprecated old predefined RC5 cipher types.
```
Comment (tarcieri) [inline rc5/CHANGELOG.md:14]: We put newest versions first in changelogs, and also generally separate release PRs from code change PRs.

```suggestion
## Unreleased
- Added u8 and u128 word size support.
- Enabled custom word size, key size, and round count values.
- Deprecated old predefined RC5 cipher types.

## 0.0.1 (2023-02-10)
- Initial release
```

File: rc5/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rc5"
-version = "0.0.1"
+version = "0.1.0"
```
Comment (tarcieri) [inline rc5/Cargo.toml:3]:
```suggestion
version = "0.1.0-pre"
```

**Comment by NamedNeon** [comment]
Hi, it has been a few months so I figured I'd just leave a message here to see if anything further is needed from me before this can be reviewed and hopefully merged.

**Review by tarcieri — APPROVED** [review]
Thanks!

**Pull request merged**